sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/Spark33XShims.scala

```diff
@@ -312,6 +312,39 @@ trait Spark33XShims extends Spark321PlusShims with Spark320PlusNonDBShims {
         (a, conf, p, r) => new BinaryAstExprMeta[LessThanOrEqual](a, conf, p, r) {
           override def convertToGpu(lhs: Expression, rhs: Expression): GpuExpression =
             GpuLessThanOrEqual(lhs, rhs)
+        }),
+      GpuOverrides.expr[Alias](
```

Comment:
This should leverage the new `TypeSig` interval type shims added in #5020 to avoid copying a lot of code just to override the accepted types for Spark 3.3

Reply:
Updated